front/app/containers/Admin/projects/project/analysis/Insights/ReferenceLink.tsx

```diff
@@ -31,6 +31,39 @@ const StyledLink = styled(Link)<{ isActive: boolean }>`
     }`}
 `;
 
+const referenceDisplayValue = (input, customField, localize) => {
+  const titleMultiloc = input?.data.attributes.title_multiloc;
+  if (titleMultiloc && Object.keys(titleMultiloc).length > 0) {
+    return localize(titleMultiloc);
+  }
+
+  const customFieldKey = customField?.data.attributes.key;
+  if (!customFieldKey) {
+    return null;
+  }
+  if (customField?.data.attributes.input_type === 'select') {
+    if (
+      input?.data.attributes.custom_field_values?.[customFieldKey] === 'other'
```

Comment:
Ideally, if the chosen option is not "other", I would like to show the title(s) of the selected options. Right now we would show the key of the option(s) instead. It seems complicated, as we need to fetch the options, and we could e.g. have multiple selected choices with one of them being "other". Today, we don't show the non-other options in the results and summary, so it's not really a problem. Would this be worth the extra effort?

Reply:
> Today, we don't show the non-other options in the results and summary, so it's not really a problem

@sebastienhoorens Is this true? I generated a summary for a select question "Other" response, and I see the AI summary references the non-other option of "Community Garden" :thinking: Or maybe I'm misunderstanding what you're saying? :)

![image](https://github.com/user-attachments/assets/186354b3-d8ff-4b89-acc5-36eb24a1dff6)

Reply:
NOTE: Didn't happen when summary was auto-generated once we had more than 10 inputs. Then it works correctly + applies correct filter.